assets/queries/cloudFormation/ec2_network_ACL_deny_rules/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ec2_network_ACL_denied_traffic",
+  "queryName": "EC2 network ACL denied traffic",
```

Comment:
please change the **DIRECTORY name** and id to an all-lowercase string matching the query name: `ec2_network_acl_ineffective_denied_traffic`

change query name to: `EC2 Network ACL Ineffective Denied Traffic`